server/src/test/java/org/elasticsearch/index/mapper/TypeParsersTests.java

```diff
@@ -232,55 +235,49 @@ public TokenStream create(TokenStream tokenStream) {
     }
 
     public void testParseMeta() {
-        FieldMapper.Builder<?> builder = new KeywordFieldMapper.Builder("foo");
-        Mapper.TypeParser.ParserContext parserContext = new Mapper.TypeParser.ParserContext(null, null, null, null, null, null);
-
         {
-            Map<String, Object> mapping = new HashMap<>(Map.of("meta", 3));
             MapperParsingException e = expectThrows(MapperParsingException.class,
-                    () -> TypeParsers.parseField(builder, builder.name, mapping, parserContext));
```

Comment:
KeywordFieldMapper.Builder doesn't match the signature of `parseField` anymore (because we don't actually use it in parsing), so I've adjusted these to directly test the `parseMeta` method.